sw/device/silicon_creator/lib/drivers/otp.h

```diff
@@ -43,12 +43,9 @@ uint64_t otp_read64(uint32_t address);
  *
  * @param address The address to read from offset from the start of OTP memory.
  * @param data The output buffer of at least length `len`.
- * @param len The number of bytes to read from OTP.
- * @return `kErrorOtpBadAlignment` if `address` or `address` + `len` are
- * misaligned, `kErrorOk` otherwise.
+ * @param len The number of 32-bit words to read from OTP.
```

Comment:
nit: `len` -> `num_words`. Also, the comment above should be updated (still mentions bytes and alignment).